tests/lib/test-utils.js

```diff
@@ -13,7 +13,9 @@ module.exports = {
     if (global.location) {
       return urlUtils.getOrigin(global.location.href);
     }
-    return 'http://localhost:8081';
+    return /^v0\.(?:8|10)/.test(process.version)
```

Comment:
The server is started without a specific `hostname` https://github.com/sockjs/sockjs-client/blob/3a93e2ea10389709f2d51c3352202e6efccf431b/tests/support/sockjs_server.js#L76, so it binds to `0.0.0.0` in node `>=0.8  <=0.10` and `::` in node `>0.10`.

Here we check the node version and we make sure to use a literal IPv6 address for node `>0.10`.